fix: sandbox sed commands, style/clarity improvements

HastD opened the pull request:
The GNU sed program is capable of reading and writing files via the r/w commands and executing external programs via the e command. To eliminate the possibility of command injection, any sed usage where the script to be run has variable substitutions should use the `--sandbox` option to ensure scripts attempting to use the e/w/r commands are rejected.

Also made some conciseness/style/clarity improvements to various sed scripts.

RoyalOughtness commented on files/justfiles/toggles.just:
```diff
@@ -61,19 +61,19 @@ toggle-bluetooth-modules:
+    KDE_GLOBALS_FILE='/etc/xdg/kdeglobals'
+    if test -e "$KDE_GLOBALS_FILE"; then
+      if grep -q '^ghns=false' "$KDE_GLOBALS_FILE"; then
+        sed -i 's/^ghns=.*/ghns=true/' "$KDE_GLOBALS_FILE"
```
shouldn't this fail if the original value was something other than false?

otherwise we're paving over undefined behavior

HastD replied:
We do check right above that the file has a line starting with `ghns=false`, but you're probably right it's best to have the same check line-by-line in `sed`. The idea with the `.*` at the end is just to make sure it's replacing the whole rest of the line and not getting tripped up e.g. if there's extra whitespace at the end of the line or something.

RoyalOughtness replied:
what you said makes sense and what you had before is fine then imo

HastD committed:
fix: sandbox sed commands, style/clarity improvements

The GNU sed program is capable of reading and writing files and
executing external programs. To eliminate the possibility of command
injection, any sed usage where the script to be run has variable
substitutions should use the `--sandbox` option to ensure scripts
attempting to use the e/w/r commands are rejected.

Also made some conciseness/style/clarity improvements to various sed
scripts.

Signed-off-by: Daniel Hast <hast.daniel@protonmail.com>
